src/modules/rlm_python3/configure.ac

```diff
@@ -58,7 +59,13 @@ if test x$with_[]modname != xno; then
 			'`
 		AC_MSG_NOTICE([Sanitized cflags were \"${mod_cflags}\"])
 
-		python3_ldflags=`${PYTHON3_CONFIG_BIN} --ldflags`
+		dnl # From python 3.8, --embed is required
+		dnl # https://bugs.python.org/issue36721
+		if test x"$PYTHON_VERSION" = x"3.8"; then
```

Comment:
So we want it to break again with 3.9?

Reply:
Well, I didn't know whether that was a change for good, or just for this version.

Reply:
The comment states `From python 3.8, --embed is required`

Reply:
My bad. Probably I meant "For Python 3.8" on that comment.
Having said that, it seems the change is here to stay

Reply:
Comparing version strings can be tricky. There is this macro AX_COMPARE_VERSION from the autoconf-archive. Is it ok to use those?

Reply:
Yeah that's fine